packages/contractkit/src/wallets/rpc-wallet.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  ensureLeading0x,
+  normalizeAddressWith0x,
+  privateKeyToAddress,
+} from '@celo/utils/lib/address'
+import { EncodedTransaction, provider, Tx } from 'web3-core'
+import { DefaultRpcCaller, RpcCaller } from '../utils/rpc-caller'
+import { encodeTransaction, rlpEncodedTx } from '../utils/signing-utils'
+import { RpcSigner } from './signers/rpc-signer'
+import { WalletBase } from './wallet'
+
+/*
+ *   WARNING: This class should only be used with well-permissioned providers (ie IPC)
+ *   to avoid sensitive user 'privateKey' and 'passphrase' information being exposed
+ */
+export class RpcWallet extends WalletBase {
+  protected readonly rpc: RpcCaller
+
+  constructor(protected _provider: provider) {
+    super()
+    this.rpc = new DefaultRpcCaller(_provider)
+  }
+
+  async addAccount(privateKey: string, passphrase: string): Promise<string> {
+    privateKey = ensureLeading0x(privateKey)
```

Comment:
c'mon man. new variable

Reply:
I totally agree with this but this is present all over contractkit utils functions and I think I borrowed it...